lib/matplotlib/colors.py

```diff
@@ -1169,6 +1169,82 @@ class DivergingNorm(TwoSlopeNorm):
     ...
 
 
+class SymNorm(Normalize):
+    def __init__(self, vcenter=0.0, vmax=None, clip=False):
```

Comment:
```suggestion
    def __init__(self, vcenter=0, vmax=None, clip=False):
```
Is sufficient due to automatic casting.